code/modules/ruins/mystery_box.dm

```diff
@@ -0,0 +1,290 @@
+			new /obj/item/weapon/weldingtool(src)
+			new /obj/item/weapon/hatchet(src)
+			new /obj/item/weapon/crowbar(src)
+		if(11 to 15)
+			new /obj/item/weapon/reagent_containers/glass/beaker/bluespace(src)
+		if(16 to 20)
+			for(var/i in 1 to 10)
+				new /obj/item/weapon/ore/diamond(src)
+		if(21 to 25)
+			for(var/i in 1 to 5)
+				new /obj/item/weapon/poster/contraband(src)
+		if(26 to 30)
+			for(var/i in 1 to 3)
+				new /obj/item/weapon/reagent_containers/glass/beaker/noreact(src)
+		if(31 to 35)
+			new /obj/item/seeds/cash(src)
+		if(36 to 40)
+			new /obj/item/weapon/melee/baton(src)
+		if(41 to 45)
+			new /obj/item/clothing/under/shorts/red(src)
+			new /obj/item/clothing/under/shorts/blue(src)
+		if(46 to 50)
+			new /obj/item/clothing/under/chameleon(src)
+			for(var/i in 1 to 7)
+				new /obj/item/clothing/tie/horrible(src)
+		if(51 to 52) // 2% chance
+			new /obj/item/weapon/melee/classic_baton(src)
+		if(53 to 54)
+			new /obj/item/toy/balloon(src)
+		if(55 to 56)
+			var/newitem = pick(subtypesof(/obj/item/toy/prize))
+			new newitem(src)
+		if(57 to 58)
+			new /obj/item/toy/syndicateballoon(src)
+		if(59 to 60)
+			new /obj/item/weapon/gun/energy/kinetic_accelerator/hyper(src)
+			new /obj/item/clothing/suit/space(src)
+			new /obj/item/clothing/head/helmet/space(src)
+		if(61 to 62)
+			for(var/i in 1 to 5)
+				new /obj/item/clothing/head/kitty(src)
+				new /obj/item/clothing/tie/petcollar(src)
+		if(63 to 64)
+			for(var/i in 1 to rand(4, 7))
+				var/newcoin = pick(/obj/item/weapon/coin/silver, /obj/item/weapon/coin/silver, /obj/item/weapon/coin/silver, /obj/item/weapon/coin/iron, /obj/item/weapon/coin/iron, /obj/item/weapon/coin/iron, /obj/item/weapon/coin/gold, /obj/item/weapon/coin/diamond, /obj/item/weapon/coin/plasma, /obj/item/weapon/coin/uranium)
+				new newcoin(src)
+		if(65)
+			new /obj/item/clothing/suit/ianshirt(src)
+			new /obj/item/clothing/suit/hooded/ian_costume(src)
+		if(66)
+			// don't worry, even if you fuck up and fail to unlock the
+			// case, the immortal candy bar will tp somewhere else
+			new /obj/item/weapon/reagent_containers/food/snacks/candy/youtried
+		if(67 to 68)
+			for(var/i in 1 to rand(4, 7))
+				var /newitem = pick(subtypesof(/obj/item/weapon/stock_parts) - /obj/item/weapon/stock_parts/subspace)
+				new newitem(src)
+		if(69 to 70)
+			for(var/i in 1 to 5)
+				new /obj/item/weapon/ore/bluespace_crystal(src)
+		if(71 to 72)
+			new /obj/item/weapon/pickaxe/drill(src)
+		if(73 to 74)
+			new /obj/item/weapon/pickaxe/drill/jackhammer(src)
+		if(75 to 76)
+			new /obj/item/weapon/pickaxe/diamond(src)
+		if(77 to 78)
+			new /obj/item/weapon/pickaxe/drill/diamonddrill(src)
+		if(79 to 80)
+			new /obj/item/weapon/cane(src)
+			new /obj/item/clothing/head/collectable/tophat(src)
+		if(81 to 82)
+			new /obj/item/weapon/gun/energy/plasmacutter(src)
+		if(83 to 84)
+			new /obj/item/toy/katana(src)
+		if(85 to 86)
+			new /obj/item/weapon/defibrillator/compact(src)
```

Comment:
That is not a big deal. So, a traitor who just so happens to have it as an objective, assuming he even *cares* about his objective is much more apt to go hunt down the one he knows exists rather than scour maint for a 2% roll.

Even if he did roll it, the Traitor would probably not care much.